### PR TITLE
Update issue templates

### LINKED ISSUE #225: Issue template
It might be nice to have some sort of template for issues. 
Just to make people have to think about what their latest version is.
Something like the following, perhaps?

> In general, please make sure you are using the latest versions of software when reporting an issue.
> Please include the result of the following commands, which will print out the version of python, ipympl and the jupyter ecosystem.
> ```
> python -c "import sys; print(sys.version); import ipympl; print('ipympl version:', ipympl.__version__)" & jupyter --version
> ```
> \`\`\`
> Paste here.
> \`\`\`

This gives the following output:
```python
3.8.2 | packaged by conda-forge | (default, Mar 23 2020, 17:32:17) [MSC v.1916 64 bit (AMD64)]
ipympl version: 0.5.6
jupyter core     : 4.6.3
jupyter-notebook : 6.0.3
qtconsole        : 4.7.2
ipython          : 7.13.0
ipykernel        : 5.2.0
jupyter client   : 6.1.3
jupyter lab      : 2.1.0
nbconvert        : 5.6.1
ipywidgets       : 7.5.1
nbformat         : 5.0.4
traitlets        : 4.3.3
```

### PR DESCRIPTION
**Pull request by martinRenou**
Should fix #225

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+## Versions
+<!--
+In general, please make sure you are using the latest versions of software when reporting an issue. If you are not using the latest JupyterLab, please be sure to install the right labextension: https://github.com/matplotlib/ipympl#install-an-old-jupyterlab-extension
+
+Please include the result of the following command, which will print out the version of python, ipympl and the Jupyter ecosystem.
+
+```bash
+python -c "import sys; print(sys.version); import ipympl; print('ipympl version:', ipympl.__version__)" && jupyter --version && jupyter nbextension list && jupyter labextension list
```
Comment (thomasaarholt) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]:
```
python -c "import sys; print('\n',sys.version); import ipympl; print('ipympl version:', ipympl.__version__)" && jupyter --version && jupyter nbextension list && jupyter labextension list
```
I added a newline at the beginning. Makes it a little easier to copy for people who are new to command line.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Versions
```
Comment (thomasaarholt) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: Let's place `## Versions` after the ## Describe the issue bit.

**Comment by thomasaarholt** [comment]
Good idea printing the extensions! Just two minor suggestions. We could wrap the version print in a [collapsible section](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab) if you want, but I'm not sure it's necessary with the relatively low number of issues that are created here.

**Comment by martinRenou** [comment]
I am afraid putting the versions bit in a collapsible section would put too much noise next to the commands. And it might confuse the user, what do you think?

**Comment by thomasaarholt** [comment]
I agree with your assessment :)

**Comment by martinRenou** [comment]
Let's merge it then :) Thanks for the suggestion. I'm sure it's going to help a lot

**Pull request merged**